src/architecture/services.md

```diff
@@ -402,18 +402,17 @@ The service, which `tx_from_raw` method
 will be called for a particular transaction, is chosen
 based on the `service_id` field in the transaction serialization.
 
-### Genesis Block Handler
+### Initialization Handler
 
 ```rust
 use serde_json::Value;
 
-fn handle_genesis_block(&self, fork: &mut Fork)
-                        -> Value {
+fn initialize(&self, fork: &mut Fork) -> Value {
     Value::Null
 }
 ```
 
-`handle_genesis_block` returns an initial
+`initialize` returns an initial
```

Comment:
Can't add inline, but "hence the method name" no longer makes sense in https://github.com/exonum/exonum-doc/pull/130/files#diff-0dff3c1379cd0e3496d9af1f140a731bR423

Reply:
Jeez, totally missed that. Thanks for noticing!